web-test-runner.config.js

```diff
@@ -23,9 +24,21 @@ export default {
 		},
 		{
 			name: 'perf',
-			files: getPattern('perf')
+			files: getPattern('perf'),
+			concurrency: 1,
+			concurrentBrowsers: 1,
+			browsers: [playwrightLauncher({
+				async createPage({ context }) {
+					const page = await context.newPage();
+					await page.emulateMedia({ reducedMotion: 'reduce' });
+					return page;
+				}
+			})],
 		}
 	],
+	plugins: [
+		renderPerformancePlugin(),
```

Comment:
I thought it would be better to define this within the `perf` section but unfortunately some node commands don't get defined if its there.